src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/hmi/on_driver_distraction_notification.h

```diff
@@ -74,6 +74,19 @@ class OnDriverDistractionNotification
 
  private:
   DISALLOW_COPY_AND_ASSIGN(OnDriverDistractionNotification);
+
+  /**
+   * @brief Adds lock screen dismissal warning message to notification
+   * @param notification_so Notification as SmartObject smart pointer which will
+   * be contain lock screen warning message
+   * @param application_manager Reference to Application Manager instance
+   * @param app_id ID of Application, which should show lock screen warning
+   * message
+   * @return bool Returns true if lock screen message has been set into
+   * notification successfully, otherwise returns false
+   */
+  bool AddLockScreenDismissalWarningToMessage(
```

Comment:
@IGapchuk feels like duplicaiton with appManagr

Reply:
@LuxoftAKutsan Fixed in 048fe1415fce72016340c817c56367238ee052f6